DoseVolumeHistogram/Logic/vtkSlicerDoseVolumeHistogramModuleLogic.cxx

```diff
@@ -326,8 +329,24 @@ std::string vtkSlicerDoseVolumeHistogramModuleLogic::ComputeDvh(vtkMRMLDoseVolum
     vtkSegment* segment = segmentationCopy->GetSegment(*segmentIdIt);
 
     // Get segment labelmap
-    vtkOrientedImageData* segmentLabelmap = vtkOrientedImageData::SafeDownCast( segment->GetRepresentation(
+    vtkSmartPointer<vtkOrientedImageData> segmentLabelmap = vtkOrientedImageData::SafeDownCast( segment->GetRepresentation(

```

Comment:
Same here, it's an important (and somewhat dangerous) change that now the returned object needs to be owned. Must be documented well, in GetRepresentation as well.

Reply:
This one doesn't need to be owned (vtkSegment::GetRepresentation() still returns a pointer to the original representation in the segment. vtkSegmentationNode::GetBinaryLabelmap() returns a new instance.